### PR TITLE
Add support for opening cross-organisation PRs

### PR DESCRIPTION
**Pull request by JamesDawson**
Adds a new PowerShell module `Endjin.CodeOps`

### CODE REVIEW

File: Endjin.CodeOps/Endjin.CodeOps.psd1 [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'f93ff550-8cac-4de2-9ac1-c500a4bac6fb'
+
+# Author of this module
+Author = 'james'
```
Comment (HowardvanRooijen) [inline Endjin.CodeOps/Endjin.CodeOps.psd1]: Which one? :D

File: Endjin.CodeOps/Endjin.CodeOps.psd1 [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+# Company or vendor of this module
+CompanyName = 'Unknown'
```
Comment (HowardvanRooijen) [inline Endjin.CodeOps/Endjin.CodeOps.psd1]: I'm pretty sure I can guess!

File: Endjin.CodeOps/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Endjin.CodeOps
+
+This PowerShell module contains a variety functions useful for manipulating git repos and .NET solutions.
```
Comment (HowardvanRooijen) [inline Endjin.CodeOps/README.md]: Can we have a few more details!

Is there a license?

Comment (JamesDawson) [reply Endjin.CodeOps/README.md]: The repo has the standard license - module metadata is now updated and I've added a license badge

**Pull request merged**